datacenter/ucp/2.2/guides/release-notes.md

```diff
@@ -69,6 +69,15 @@ and the API is fully interactive within the UCP UI.
   is to do a page refresh which completes the configuration change.
   * Removing stacks from the UI may cause certain resources to not be deleted,
   including networks or volumes. Workaround is to delete the resources directly.
+  * When you create a network and check 'Enable hostname based routing', the web
+  UI doesn't apply the HRM labels to the network. As a workaround,
+  [create the network using the CLI](https://docs.docker.com/datacenter/ucp/2.2/guides/user/services/use-domain-names-to-access-services/#service-labels).
+  * The web UI is not persisting changes to session timeout settings. As a workaround
```

Comment:
Maybe "does not currently persist" or "cannot currently persist"? rather than "is not persisting"